resources/addresses/responses/all_addresses.yml

```diff
@@ -36,7 +36,6 @@ content:
           address_zip: 94107-1741
           address_country: UNITED STATES
           metadata: {}
-          recipient_moved: false
```

Comment:
just pointing out that removing this will make this an "incomplete" payload

Reply:
hmm but the payload will look like that (without `recipient_moved`) for almost everyone. only if you actually have ncoa enabled will you even see that field, so I'm not sure we want to include it in the regular response.